lesson_source/0028/pl.md

```diff
@@ -0,0 +1,313 @@
+        <li>Element 2
+      </ul>
+    </p>
+
+### Podstawowa struktura HTML
+
+    :::html
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="UTF-8">
+        <title>Tytuł strony</title>
+      </head>
+      <body>
+      
+      </body>
+    </html>
+
+Element `<title>` zawiera tytuł strony wyświetlany przez przeglądarkę jako nazwa karty.<br>
+Element `<body>` ma zawierać całą właściwą treść strony - na dzisiejszych zajęciach wszystko będziemy umieszczać właśnie tam.
+
+### Podstawowe elementy
+
+* `<p>Akapit</p>`
+* `<h1>Duży nagłówek</h1>`
+* `<h2>Trochę mniejszy nagłówek</h2>`
+* ...
+* `<h6>Najmniejszy nagłówek</h6>`
+* `<br>` - nowa linia (uwaga – zwykłe entery są ignorowane przez przeglądarki!)
+* `<hr>` - pozioma linia
+
+#### Lista numerowana
+
+    :::html
+    <ol>
+      <li>Element 1</li>
+      <li>Element 2</li>
+    </ol>
+
+#### Lista nienumerowana
+
+    :::html
+    <ul>
+      <li>Element 1</li>
+      <li>Element 2</li>
+    </ul>
+
+#### Obrazki
+
+    :::html
+    <img src="cat_image.jpg" width="200" height="200">
+
+* `src` - nazwa pliku z obrazkiem, musi być w tym samym folderze co plik strony
+* `width`, `height` - szerokość i wysokość obrazka (opcjonalne)
+
+#### Linki
+
+    :::html
+    <a href="https://www.python.org/">Strona Pythona</a>
+
+* `href` - adres docelowy linku. Jeśli zaczniemy go od `/` - będzie oznaczać adres w aktualnie wyświetlanej stronie
+(np. jeśli nasza strona to `www.sluchamjazzu.pl` to link o adresie `/podstrona` będzie prowadził do adresu
+`www.sluchamjazzu.pl/podstrona`).
+
+#### Tabele
+
+    :::html
+    <table border>
+      <tr>
+        <th>Imię</th>
+        <th>Nazwisko</th>
+      <tr>
+      <tr>
+        <td>Hiromi</td>
+        <td>Uehara</td>
+      <tr>
+    </table>
+
+* `<table>` - nadrzędny element oznaczający tabelę. Atrybut `border` powoduje dodanie ramki do tabeli
+* `<tr>` - wiersz tabeli
+* `<td>` - komórka tabeli
+* `<th>` - komórka nagłówkowa - od zwykłej różni się głównie pogrubioną czcionką
+
+### Pełny przykład
+
+    :::html
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Przykładowa strona</title>
+    </head>
+    <body>
+        <h1>To jest przykładowa strona</h1>
+        Ta strona zawiera:
+        <p>Przykładowy akapit</p>
+        <p>Oraz drugi akapit<br>z wstawioną nową linią w środek</p>
+        <ol>
+            <li>Listę</li>
+            <li>numerowaną</li>
+        </ol>
+        <a href="//pylove.org">Link do strony PyLove</a>
+        <table border>
+            <tr>
+                <th>Oraz</th>
+                <th>Przykładową</th>
+            </tr>
+            <tr>
+                <td>Tabelkę</td>
+                <td>z kilkoma</td>
+            </tr>
+            <tr>
+                <td>wierszami</td>
+                <td></td>
+            </tr>
+        </table>
+    </body>
+    </html>
+
+### Przykład tabeli do zadania 2.
+
+<table border>
```

Comment:
brakuje wcięciaci kodu ::hmtl

Reply:
To nie ma być kod tylko faktyczna tabela, nie wiedziałem jak opisać colspan w zrozumiały sposób więc uznałem, że wrzucę po prostu przykładową tabelę